app/models/sponsorship.rb

```diff
@@ -21,11 +21,13 @@ class Sponsorship < ActiveRecord::Base
   belongs_to :sponsor
   belongs_to :orphan
 
-  delegate :name, :additional_info, :id, to: :sponsor, prefix: true
+  delegate :name, :additional_info, to: :sponsor, prefix: true
   delegate :date_of_birth, :gender, to: :orphan, prefix: true
 
   scope :all_active, -> { where(active: true) }
   scope :all_inactive, -> { where(active: false) }
+  scope :load_with_orphans_for_sponsor,
+         ->(sponsor_id) {where("sponsor_id = ?", sponsor_id).eager_load(:orphan).all}
```

Comment:
I wonder whether `load_with_orphans_for @sponsor` would read better.
Also, a question that has come up before: Should the default scope for sponsorship eager load sponsor and orphan instead?